SPECS/kernel/config

```diff
@@ -12,6 +12,7 @@ CONFIG_LD_IS_BFD=y
 CONFIG_LD_VERSION=23700
 CONFIG_LLD_VERSION=0
 CONFIG_CC_CAN_LINK=y
+CONFIG_CC_CAN_LINK_STATIC=y
```

Comment:
This is not desired due to recent changes with glibc-static

https://github.com/microsoft/CBL-Mariner/pull/4037